src/layouts/docs/utils/_local_platform-docs-mdx/README.md

```diff
@@ -0,0 +1,6 @@
+# `@hashicorp/platform-docs-mdx`
+
+🚨 _PLEASE NOTE_: the files in this folder have been localized from our [`@hashicorp/platform-docs-mdx` package](https://github.com/hashicorp/web-platform-packages/tree/main/packages/docs-mdx). The rationale for localizing these files, rather than making upstream changes in the platform-mdx package, is:
```

Comment:
Have added this README to record the rationale for localizing this work before modifying it. Open to thoughts here 🙇

Reply:
For consideration: acknowledging that this will most likely live long-term in this repo, we could refactor it a bit to drop the dead code and call it something else?  I think it'd also be fine to defer that to not increase the scope of the PR.

Reply:
Great call, will at least drop the dead code now 👍 Going to defer the name change and keep this readme in place for now (so the origin is clear as we migrate stuff).

Reply:
✅ Dropped the dead code